apps/documentation/project.json

```diff
@@ -95,7 +93,7 @@
         "karmaConfig": "apps/documentation/karma.conf.js",
         "inlineStyleLanguage": "scss",
         "assets": ["apps/documentation/src/favicon.ico", "apps/documentation/src/assets"],
-        "styles": ["apps/documentation/src/styles.scss"],
+        "styles": [],
```

Comment:
I have removed the `apps/documentation/src/styles.scss` because the test fails with the error given below 

> ./apps/documentation/src/styles.scss - Error: Module Error (from ./node_modules/postcss-loader/dist/cjs.js):
/home/htadmin/Projects/charts-on-fhir/node_modules/material-icons/iconfont/_mixins.scss:43:4: Can't resolve '../../../node_modules/material-icons/iconfont/material-icons/iconfont/material-icons-two-tone.woff' in '/home/htadmin/Projects/charts-on-fhir/apps/documentation/src'
 
for import using path
> $material-icons-font-path: "material-icons/iconfont/" !default;
> @import "material-icons";